lib/asciidoctor-pdf/prawn_ext/extensions.rb

```diff
@@ -191,7 +191,7 @@ def register_font data
   def font name = nil, options = {}
     if name
       options = { size: options } if ::Numeric === options
-      if IconSets.include? name
+      if name != 'fa' && IconSets.include?(name)
```

Comment:
It's not clear to me why "fa" is being excluded here. Could you explain?

Reply:
It's mostly a sanity check, but I believe there may be negative outcomes if we don't exclude it here:

* FontAwesome 5 no longer ships with a base `fa.tff` (or equivalent) font file. Instead the fonts are split into multiple files based on the category (`brands`, `regular` or `solid`). So if we tried to load a base `fa` font, an exception would likely be raised.
* `Prawn::Icon` still accepts FontAwesome 4 specifiers in the format of `fa-{icon-name}` for full backwards compatibility. `Prawn::Icon` contains a compatibility shim and knows how to map these legacy icon specifiers to the corresponding FontAwesone 5 family and definition. 

An icon specifier like **`fa-automobile`** is completely valid, but it maps directly to the **`fas-car`** specifier as seen [here](https://github.com/jessedoyle/prawn-icon/blob/master/data/fonts/fa4/shims.yml#L21). This means that we actually want to load the `fas` (FontAwesome Solid) font instead.

Reply:
If I remember correctly, this block is lazy loading the font. So if we don't lazy load when "fa" is used, then I would expect the processor to crash. That is, unless Prawn::Icon is now loading behind the scenes. I'll need to do some testing.